src/main/resources/com/databricks/spark/sql/perf/mllib/config/mllib-large.yaml

```diff
@@ -59,6 +59,10 @@ benchmarks:
       optimizer:
         - em
         - online
+  - name: fpm.FPGrowth
+    params:
+      numItems: 10000
+      itemSetSize: 10
```

Comment:
This is a pretty big average itemset size.  It could be worth testing 2 sizes, say 4 and 10.  (Think of this number as being, e.g., the number of items in an online shopping cart at checkout.)